src/Files.App/ServicesImplementation/SideloadUpdateService.cs

```diff
@@ -27,8 +27,8 @@ public sealed class SideloadUpdateService : ObservableObject, IUpdateService, ID
 		[DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
 		private static extern uint RegisterApplicationRestart(string pwzCommandLine, int dwFlags);
 
-		private const string SIDELOAD_STABLE = "https://cdn.files.community/files/stable/Files.Package.appinstaller";
-		private const string SIDELOAD_PREVIEW = "https://cdn.files.community/files/preview/Files.Package.appinstaller";
+		private const string SIDELOAD_STABLE = "https://cdn.files.community/files/stable/Files.App.Package.appinstaller";
+		private const string SIDELOAD_PREVIEW = "https://cdn.files.community/files/preview/Files.App.Package.appinstaller";
```

Comment:
This isn't going to help for existing users

Reply:
Do you have any idea to fix the auto updating issue?

Reply:
We can make it work if we split it into two steps
1. Update this url and release an update with the change
2. Rename the projects in another update